country_by_country/img_table_extraction/llama_parse_extractor.py

```diff
@@ -40,13 +41,17 @@ def __init__(self, **kwargs: dict) -> None:
         You are free to define any parameter LlamaParse recognizes
         """
         self.kwargs = kwargs
+        self.type = "llama_parse"
 
         # Load LLAMA_CLOUD_API_KEY from .env file
         load_dotenv()
         # llama-parse is async-first
         nest_asyncio.apply()
```

Comment:
This should be called super early in running the program and only once.  Again seems `__main__.py` could be a good place.

Reply:
I moved env loading to `country_by_countr/__main__.py` file